src/pages/ServicePage.js

```diff
@@ -26,28 +26,84 @@ class CategoryPage extends React.Component {
     componentDidMount(): void {
         iss(`service/${this.props.params.id}/`)
             .then(data => {
-                console.log(data);
                 this.setState({
                     object: data,
                 });
             });
 
-        // FIXME: display error
+        // FIXME: display error on failure to connect
+    }
+
+    renderPhones(phones: Array<Object>): React.Element {
```

Comment:
We probably want to do something useful with the different phone types. Only show some of the phone types etc.

Phone types are `phone`, `mobile`, `fax`, `tty` and `freecall`.